### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by karimcambridge**
Fix CSS.
Remove unnecessary font.
Change example from saarp to example.org or w/e.
Spaces -> tabs.

### CODE REVIEW

File: html/dist/voiceChat.css [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
+﻿* {
     margin: 0 0;
     padding: 0 0;
     border-spacing: 0px;
     box-sizing: content-box;
 }
-@font-face {
-    font-family: "GTAV_font_6";
-    src: url('./font6.ttf') format("truetype");
-}
```
Comment (rt-2) [inline html/dist/voiceChat.css]: You told me on discord that the font was not there, but actually it IS int he dist folder

File: html/dist/voiceChat.css [hunk 2]
```diff
@@ -25,7 +20,6 @@
     position: relative;
     width: 100%;
     height: auto;
-    font-family: GTAV_font_6;
```
Comment (rt-2) [inline html/dist/voiceChat.css:28]: We should try to fix this

File: html/dist/voiceChat.css [hunk 3]
```diff
@@ -1,17 +1,12 @@
...
     box-sizing: content-box;
 }
-@font-face {
-    font-family: "GTAV_font_6";
-    src: url('./font6.ttf') format("truetype");
-}
 
 #iframes_container {
-	//display: none;
+	display: none;
```
Comment (rt-2) [inline html/dist/voiceChat.css]: https://github.com/rt-2/RageMP-SAARPV-voiceChat/blob/master/html/index.html#L107 is repeating it btw

**Review by rt-2 — COMMENTED** [review]
Nice job overall! We need to figure out what,s with the font tho...
Thank you
rt-2

File: html/dist/voiceChat.css (hunk 2 above)
Comment (karimcambridge) [reply html/dist/voiceChat.css:28]: Fixed.

File: html/dist/voiceChat.css (hunk 1 above)
Comment (karimcambridge) [reply html/dist/voiceChat.css]: Fixed.

File: html/dist/voiceChat.css (hunk 3 above)
Comment (karimcambridge) [reply html/dist/voiceChat.css]: Fixed.

**Review by rt-2 — APPROVED** [review]
Very nice job, thank you for your time Kar!

**Pull request merged**